src/core_modules/capture-core/components/Pages/EnrollmentEvent/EnrollmentEventPage.container.js

```diff
@@ -1,10 +1,34 @@
 // @flow
 import React, { useState } from 'react';
+// $FlowFixMe
+import { useSelector, shallowEqual } from 'react-redux';
+import { useProgramInfo } from '../../../hooks/useProgramInfo';
 import { pageMode } from './EnrollmentEventPage.const';
 import { EnrollmentEventPageComponent } from './EnrollmentEventPage.component';
 
 export const EnrollmentEventPage = () => {
+    const { programId, stageId } = useSelector(
+        ({
+            router: {
+                location: { query },
+            },
+        }) => ({
+            programId: query.programId,
+            stageId: query.stageId,
+        }),
+        shallowEqual,
+    );
     const [mode] = useState(pageMode.VIEW);
-
-    return <EnrollmentEventPageComponent mode={mode} />;
+    const { program } = useProgramInfo(programId);
+    const programStage = [...program.stages?.values()].find(
+        item => item.id === stageId,
+    );
+    const eventAccess = { read: true, write: true };
```

Comment:
The `eventAccess` will change once the edit/view form is implemented [DHIS2-11208](https://jira.dhis2.org/browse/DHIS2-11208)